apps/spruce/src/components/TaskReview/db.ts

```diff
@@ -0,0 +1,52 @@
+import { IDBPDatabase, openDB, DBSchema } from "idb";
+
+const DB_NAME = "spruce-db";
+const STORE_NAME = "task_reviews";
+
+// CAUTION! Updating the store's schema requires updating the DB_VERSION and writing an IndexedDB migration.
```

Comment:
Can you please link a doc on how to do a migration (Doesn't need to be a custom one just some external documentation on how to do this if we need to in the future)